src/package/core/core.ts

```
@@ -859,6 +891,53 @@ export class Core {
+        const payload: DebugDrawPayload = message.payload;
+        const target = this.players.find(player => player.id === payload.player);
+        if (target) {
+          target.draw();
+          this.room.sync();
+        }
+        break;
+      }
+      case 'DebugMake': {
+        const payload: DebugMakePayload = message.payload;
+        const target = this.players.find(player => player.id === payload.player);
+        const master = catalog.get(payload.catalogId);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)